minecraft_servers/combatreforged/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "server_name": "combatreforged",
+  "nice_name": "CombatReforged",
+  "direct_ip": "combatreforged.com",
+  "server_wildcards": [
+    "%.combatreforged.com"
+  ],
+  "supported_languages": [
+     "en"
+  ],
+  "social": {
+    "web": "https://venade.net",
+    "twitter": "venade_net",
+    "discord": "https://discord.gg/venade",
+    "youtube": "https://www.youtube.com/@venadenetwork"
```

Comment:
Why? Is venade = combatreforged?

Reply:
Because CombatReforged is a service that belongs to Venade and is using the same socials and Discord server.